### PR TITLE
update thrift files and add new functions for resultSet

### PR DESCRIPTION
**Pull request by Aiee**
1. update thrift files to V2.0.0-beta
2. add AsStringTable() for resultSet which returns a 2D array of strings representing the query result

### CODE REVIEW

**Review by jievince — DISMISSED** [review]

File: go/result_set.go [hunk 1]
```diff
@@ -237,6 +237,16 @@ func (res ResultSet) GetRowValuesByIndex(index int) (*Record, error) {
...
 }
 
+// Returns the number of total rows
+func (res ResultSet) GetRowSize() int {
+	return len(res.resp.Data.Rows)
+}
+
+// Returns the number of total columns
+func (res ResultSet) GetColSize() int {
+	return len(res.resp.Data.ColumnNames)
```
Comment (laura-ding) [inline go/result_set.go]: check data is not null

File: go/result_set.go [hunk 2]
```diff
@@ -237,6 +237,16 @@ func (res ResultSet) GetRowValuesByIndex(index int) (*Record, error) {
 	}, nil
 }
 
+// Returns the number of total rows
+func (res ResultSet) GetRowSize() int {
+	return len(res.resp.Data.Rows)
```
Comment (laura-ding) [inline go/result_set.go]: check data is not null

File: go/result_set.go [hunk 3]
```diff
@@ -237,6 +237,16 @@ func (res ResultSet) GetRowValuesByIndex(index int) (*Record, error) {
 	}, nil
 }
 
+// Returns the number of total rows
+func (res ResultSet) GetRowSize() int {
```
Comment (laura-ding) [inline go/result_set.go]: loss is_empty()

Comment (Aiee) [reply go/result_set.go]: fixed

File: go/result_set.go [hunk 4]
```diff
@@ -268,12 +302,43 @@ func (res ResultSet) GetErrorCode() ErrorCode {
...
+func (res ResultSet) GetComment() string {
+	if res.resp.Comment == nil {
+		return ""
 	}
-	return res.resp.ErrorMsg
+	return string(res.resp.Comment)
+}
+
+func (res ResultSet) IsEmpty() bool {
+	if res.resp.Data == nil || res.resp.Data.ColumnNames == nil || res.resp.Data.Rows == nil {
```
Comment (laura-ding) [inline go/result_set.go]: if res.resp.Data is not null, res.resp.Data.ColumnNames and res.resp.Data.Rows is not null ,
`if res.resp.Data == nil || len(res.resp.Data.Rows) == 0`

**Review by jievince — DISMISSED** [review]

**Review by jievince — APPROVED** [review]

**Review by laura-ding — APPROVED** [review]

**Pull request merged**